pkg/agent/baker.go

```diff
@@ -1166,6 +1166,10 @@ root = "{{GetDataDir}}"{{- end}}
     {{- if TeleportEnabled }}
     snapshotter = "teleportd"
     disable_snapshot_annotations = false
+    {{- end}}
+	{{- if IsArtifactStreamingEnabled }}
```

Comment:
extra indent here?

Reply:
yes, tabs vs spaces haha, so it looked the same on VS code